cache

```diff
@@ -186,6 +186,12 @@ cache::autorestore() {
   for lockfile in ${LOCKFILES[@]}; do
     if [[ -f $lockfile ]]; then
         case $lockfile in
+          ".nvmrc")
+            cache::log
+            cache::log "* Detected $lockfile."
+            cache::log "* Fetching '~/.nvm' directory with cache keys 'nvm-${SEMAPHORE_GIT_BRANCH}-$(checksum .nvmrc),nvm-${SEMAPHORE_GIT_BRANCH},nvm-master'."
+            SEMAPHORE_LOCAL_CACHE_PATHS="$HOME/.nvm"
+            cache::restore nvm-${SEMAPHORE_GIT_BRANCH}-$(checksum .nvmrc),nvm-${SEMAPHORE_GIT_BRANCH},nvm-master
           "Gemfile.lock")
```

Comment:
cache::autosotre() part is missing please have a look at:
https://github.com/semaphoreci/toolbox/blob/master/cache#L62